recipes/ligo-remote-epics-scripts/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "ligo-remote-epics-scripts" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/ligo-remote-epics-scripts/-/archive/v{{ version }}/ligo-remote-epics-scripts-v{{ version }}.tar.gz
+  sha256: 6a40f38c1c6ccdeee646caf1ca34399d1a302db942e7cfcb110fa79f6631556e
+
+build:
+  number: 0
+  skip: true  # [win]
```

`ligo-medm` isn't being build for macOS

https://github.com/conda-forge/ligo-medm-feedstock/blob/0470df01f78b7ee69333ed78c4d346c6b990e0a2/recipe/meta.yaml#L27


```suggestion
  skip: true  # [win or osx]
```